Frameworks/CoreText/CTFramesetter.mm

```diff
@@ -48,8 +48,10 @@ CTFrameRef CTFramesetterCreateFrame(CTFramesetterRef framesetterRef, CFRange ran
         range.length = [typesetter->_string length] - range.location;
     }
 
-    StrongId<_CTFrame> ret = _DWriteGetFrame(static_cast<CFAttributedStringRef>(typesetter->_attributedString.get()), range, frameRect);
-    ret->_path.reset(CGPathRetain(path));
+    woc::StrongCF<__CTFrame*> ret;
+    ret =
```

Comment:
>ret [](start = 4, length = 3)

collapse the declaration and assignment into initialization? there is no need to declare and then assign.